grammar.js

```diff
@@ -101,8 +109,7 @@ module.exports = grammar({
       field("value", $._keybind_value),
       newline,
     ),
-    // TODO: Handle quoted values
-    _keybind_value: $ => $.keybind_value,
+    _keybind_value: $ => choice($.keybind_value, alias("clear", $.value), alias($.string_literal, $.value)),
```

Comment:
```suggestion
    _keybind_value: $ => choice($.keybind_value, alias("clear", $.keybind_clear_keyword), alias($.string_literal, $.value)),
```

could be helpful to query for (e.g. capture as a highlight group)